addons/binding/org.openhab.binding.opendaikin/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,29 @@
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: OpenDaikin Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.opendaikin;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Export-Package: 
```

Comment:
I almost forgot: It's actually not necessary to export anything. Therefor the policy was changed in our code(reviews) and this export part should be removed.
Part of this is to also move everything to internal packages. So please could you move everything from `org.openhab.binding.opendaikin` to `org.openhab.binding.opendaikin.internal` (same for handler move it inside the internal package -> ` org.openhab.binding.opendaikin.internal.handler`.)